acceptance/Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gem "mixlib-install", github: "chef/mixlib-install"
+gem "test-kitchen", github: "sersut/test-kitchen", branch: "sersut/mixlib-install-update"
+gem "chef-acceptance", github: "chef/chef-acceptance"
+
+group(:development) do
```

Comment:
I am thinking we can stop using this gem group now. I believe it's copy/paste remnants.